Library/Homebrew/dev-cmd/bump-cask-pr.rb

```diff
@@ -9,6 +9,38 @@
 module Homebrew
   module_function
 
+  class NewVersion
+    attr_reader :arm, :general, :intel
+
+    sig { params(general: T.nilable(String), arm: T.nilable(String), intel: T.nilable(String)).void }
+    def initialize(general: nil, arm: nil, intel: nil)
+      @general = parse_new_version(general) unless general.nil?
+      @intel = parse_new_version(intel) unless intel.nil?
+      @arm = parse_new_version(arm) unless arm.nil?
+
+      if @general.nil?
+        raise UsageError, "`--version-intel` must not be empty." if intel.nil?
+        raise UsageError, "`--version-arm` must not be empty." if arm.nil?
+      elsif @general && (@arm || @intel)
+        raise UsageError, "You cannot specify --version with --version-intel and --version-arm."
+      end
+    end
+
+    sig { params(version: String).returns(T.nilable(Cask::DSL::Version)) }
+    def parse_new_version(version)
+      if %w[latest :latest].include?(version)
```

Comment:
```suggestion
      if version == "latest"
```

Given the `String` type constraint, this should never be a `Symbol` anyways.